src/api/extension/wallet.js

```diff
@@ -496,3 +496,155 @@ export const withdrawalTx = async (account, delegation, protocolParameters) => {
 
   return transaction;
 };
+
+export const undelegateTx = async (account, delegation, protocolParameters) => {
+  await Loader.load();
+  const utxos = await getUtxos();
+  //estimated max multiasset size 5848
+  //estimated max value size 5860
+  //estimated max utxo size 5980
+  const MULTIASSET_SIZE = 5848;
+  const VALUE_SIZE = 5860;
```

Comment:
Hello! I’m new to Cardano. Why are these values constants? Will neither ever change? I want to learn!

Reply:
These values will be removed soon, they have no purpose here anymore. But yes these values could change depending on cardano protocol parameters.

Reply:
ok thank you!